src/Component/Symfony/Routing/Factory/BulkOperationRoutePathFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Routing\Factory;
+
+use Sylius\Component\Resource\Metadata\BulkOperationInterface;
+use Sylius\Component\Resource\Metadata\HttpOperation;
+
+class BulkOperationRoutePathFactory implements OperationRoutePathFactoryInterface
```

Comment:
Let's finalise all these small classes
```suggestion
final class BulkOperationRoutePathFactory implements OperationRoutePathFactoryInterface
```